src/lib/djerba/extract/report_to_json.py

```diff
@@ -426,15 +426,6 @@ def get_cytoband(self, gene_name):
             self.logger.warn("Unknown cytoband for gene '{0}'".format(gene_name))
         return cytoband
 
-    def image_to_json_string(self, image_path, image_type='jpeg'):
```

Comment:
This function is obsolete -- it was not being used anywhere, but left over from a previous implementation